mdk/src/main/resources/META-INF/mods.toml

```diff
@@ -62,3 +62,9 @@ description='''${mod_description}'''
     versionRange="${minecraft_version_range}"
     ordering="NONE"
     side="BOTH"
+
+# Features are specific properties of the game environment, that you may want to declare you require. This example declares
+# that your mod requires GL version 3.2 or higher. Other features will be added. They are side aware so declaring this won't
```

Comment:
Is there a file/block of code anywhere that has all the features so that we could include a link/reference so modders can check what features actually exist and they can use?

Reply:
No, not yet. It's very much an incubating "feature" at present. LOL I think it's something that could be useful for various purposes - optifine detection, java versioning, gl stuffs, etc etc. The openGLVersion is just the first "toe in the water"